### ISSUE TITLE
Contact List

### ISSUE THREAD

**Issue by adamisch** [acf-quizbowl/acf-quizbowl.github.io#4]
We should find a place to embed this, responsibly:

https://docs.google.com/spreadsheets/d/15wbthPYr4wgTVrsivOn7VFty8JsCc6zgM6z6lg268ow/edit?usp=sharing

**Comment by adamisch** [comment]
@ryanrosenberg  bump

**Comment by ryanrosenberg** [comment]
Hm, how are you envisioning the "responsibly" part of this? I can embed the sheet as view-only, although ofc any useful addition of the sheet will have everyone's email public.

**Comment by adamisch** [comment]
Yeah, view only. Not sure if bots would be able to trawl an embedded gdoc
for emails?
